### ISSUE TITLE
You can use your sniper rifle zoom in conjunction with advanced camera consoles to have a huge viewrange in camera view.

### ISSUE THREAD

**Issue by WJohn** [tgstation/tgstation#34262]
What it says on the tin.

Also really just a continuation of the bug where you can use rifle zoom without the gun in your hands.

**Comment by xxalpha** [comment]
You're just pointing the scope at the screen, makes total sense.

**Comment by praisenarsie** [comment]
@kevinz000 Weapons to surpass Plasma Gear

**Comment by Iatots** [comment]
Yeah but is hanging a magnifying lens on your monitor or putting a mark with a sharpie in the center cheating whilst playing CS:GO?

**Comment by ma44** [comment]
The more view range you have in game, the less view range you have when using the camera console.

**Comment by Sealed101** [comment]
sniper rifles have been changed in #66218, not sure if they can interact with the camera console like this

**Comment by LemonInTheDark** [comment]
Partially handled by https://github.com/tgstation/tgstation/pull/51208
It's still possible to offset your view though, since that isn't packed into view data. Would be good to fix that

**Comment by LemonInTheDark** [comment]
why?

**Comment by necromanceranne** [comment]
I figured we had https://github.com/tgstation/tgstation/issues/42567 which links to this one to also track this issue, but if you'd rather this still be open that's cool.

**Comment by LemonInTheDark** [comment]
If you're ok with it I'd like to retain this, since it reminds me of the issue more clearly then the other. if that's just me you can close this I might just have brainworms
